src/lib/app.ts

```diff
@@ -126,9 +126,10 @@ export default function getApp(
         res.send(indexHTML);
     });
 
-    app.get('*', (req, res) => {
+    app.get(`${baseUriPath}*`, (req, res) => {
```

Comment:
we only want to catch anything which starts with our `baseUriPath`!

Reply:
Makes sense. Our basepath format is `/hosted` without trailing slash and according to react-router that is the format they also expect. 

```
The base URL for all locations. If your app is served from a sub-directory on your server, you’ll want to set this to the sub-directory. A properly formatted basename should have a leading slash, but no trailing slash.
```

If we enforce that format this should be: 
```suggestion
    app.get(`${baseUriPath}/*`, (req, res) => {
```

Now that I think about it we might want to run the basename through a formatter to enforce a format we can trust. This might be too brittle for a configurable property.

Reply:
I think you are right.